src/lib.js

```diff
@@ -62,6 +80,8 @@ export function createMapping({ components, module, mapping, filter }) {
     }
   });
 
+  generateDTS(usedComponents, dts)
```

Comment:
Finally got some time to review this PR!

It would be better to handle the situation where `dts` is not provided